src/SDKs/SecurityCenter/Management.SecurityCenter/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure Security Center Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Security Center Resources.")]
+
+[assembly: AssemblyVersion("0.1.0.0")]
+[assembly: AssemblyFileVersion("0.1.0.0")]
```

Comment:
@chlahav update assemblyversion/fileversion to 0.9.0.0

Reply:
@shahabhijeet done